modules/core/src/test/java/org/apache/ignite/internal/ThreadNameValidationTest.java

```diff
@@ -0,0 +1,174 @@
+/*
```

Comment:
Please overview the whole of this file and add the spaces where it is necessary because right now there are a lot of lines stuck together which are not matched to our code style.